spartan/testnet-runbook.md

```diff
@@ -0,0 +1,116 @@
+
+## Releases
+
+The engineering team's testnet responsibilities begin after code-freeze. Here are the primary tasks:
+
+1. Confirm with engineering and product teams that all required PRs are merged
+2. Create a release branch (eg: `<repository>-v<major>.<minor>.<patch>`, e.g., `aztec-packages-v0.62.0`)
+3. Cherry-pick bug-fixes into the release branch for bugs discovered during release testing.
+4. Initiate a final build by pushing an empty commit into the release branch to trigger the `release-please` CI workflow.
+
+### Release Notes and Artifact Builds
+
+Verify the `release-please` CI workflow completed successfully and that release notes have been published.
+A successful CI run publishes the following Barretenberg artifacts with the release notes:
+
+- Barretenberg for Mac (x86 64-bit)
+- Barretenberg for Mac (Arm 64-bit)
+- Barretenberg for Linux (x86 64-bit)
+- Barretenberg for WASM
+
+Additionally, the following NPM packages are published:
+
+- BB.js
+- l1-contracts
+- yarn-project (see [publish_npm.sh](https://github.com/AztecProtocol/aztec-packages/blob/aztec-packages-v0.63.0/yarn-project/publish_npm.sh))
+
+The following Docker containers are also published:
+
+- aztecprotocol/aztec:latest
+- aztecprotocol/aztec-nargo:latest
+- aztecprotocol/cli-wallet:latest
+
+Lastly, any changes made to developer documentation are published to <https://docs.aztec.network>
+
+## Deployment
+
+After cutting a release, deploy a testnet (typically with 48 validators) using the new Docker containers. Verbose logging on Aztec nodes should be enabled by default using the following `ENV VARS`:
+
+- `LOG_JSON=1`
+- `LOG_LEVEL=debug`
+- `DEBUG=discv5*,aztec:*,-aztec:avm_simulator*,-aztec:circuits:artifact_hash,-json-rpc*,-aztec:world-state:database,-aztec:l2_block_stream*`
+
+Deployments are initiated from CI by manually running the (_name pending_) workflow.
+
+### Sanity Check
+
+After testnet deployment, perform these sanity checks (these items can also be script automated):
+
+1. Monitor for crashes and network-level health:
+   - Review testnet dashboard at `https://grafana.aztec.network/` to confirm node uptime and block production
+   - Verify overall TPS performance
+   - Create Github issues for new crash scenarios
+
+2. Spot check pod logs for component health:
+   - Tx gossiping (Bot: `Generated IVC proof`)
+   - Peer discovery (Validator (failure case): `Failed FINDNODE request`)
+   - Block proposal (Validator: `Can propose block`)
+   - Block processing (Validator: `l2BlockSourceHash`)
+   - Block proving (Prover: `Processed 1 new L2 blocks`)
+   - Epoch proving (Prover: `Submitted proof for epoch`)
+
+3. Test external node connection and sync
```

Comment:
@aminsammara this would be in additional to a 3 epochs test in QA. It is a check that the deployment itself was healthy. We want to know this as soon as the deployment completes, and before the connection information is shared with others.